stable/redis-ha/values.yaml

```diff
@@ -15,33 +15,14 @@ resources:
     limits:
       memory: 200Mi
 
-## Node labels, tolerations, and affinities for pod assignment
+## Node labels and tolerations for pod assignment
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
-## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 nodeSelector: {}
 tolerations: []
-sentinelAffinity: |
```

Comment:
Curious, is there a specific reason you removed the explicit hard anti affinity?

Reply:
I personally did not...I copied most the pull request from @bzumhagan and made some fixes...so he likely removed it